app/src/main/java/app/accrescent/client/ui/MainActivity.kt

```diff
@@ -109,21 +97,27 @@ class MainActivity : ComponentActivity() {
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun MainContent(
-    systemUiController: SystemUiController,
-    appId: String?
+    appId: String?,
+    modifier: Modifier = Modifier
```

Comment:
nit: Could you move the modifier parameter addition to a separate commit and update the lint baseline in that commit to remove this instance?

Reply:
Addition has been reverted, better fit for more in depth linting.